website/src/pages/index.module.css

```diff
@@ -0,0 +1,250 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+@import url('https://db.onlinewebfonts.com/c/00d5a78493aed4b11e2584ad7cceee49?family=Gilroy-ExtraBold');
+
+html {
+  background-color: var(--ifm-background-color);
+}
+
+html[data-theme='dark'] {
+  --ifm-background-color: #3B4252;
+  --ifm-hero-background-color: #2E3440;
+}
+html[data-theme='light'] {
+  --ifm-background-color: white;
+  --ifm-hero-background-color: whitesmoke;
+}
```

Comment:
If you're applying the Nord colour palette can you please do so consistently. Here are all of the colours (i.e. darker tones to use instead of the defaults to contrast against the background)

Also can you make sure we're using purple accents instead of the default green?

Reply:
i will try to replace the colors